dinstaller-lib/src/software.rs

```diff
@@ -7,24 +9,40 @@ pub struct Product {
     pub description: String,
 }
 
-pub fn products() -> Vec<Product> {
-    vec![
-        Product {
-            id: "Leap".to_string(),
-            name: "openSUSE Leap 15.4".to_string(),
-            description: "Leap uses source from SUSE Linux Enterprise (SLE), which...".to_string(),
-        },
-        Product {
-            id: "ALP".to_string(),
-            name: "SUSE ALP ContainerHost OS".to_string(),
-            description: "'The Adaptable Linux Platform (ALP), the next generation of Linux..."
-                .to_string(),
-        },
-        Product {
-            id: "Tumbleweed".to_string(),
-            name: "openSUSE Tumbleweed".to_string(),
-            description: "The Tumbleweed distribution is a pure rolling release version..."
-                .to_string(),
-        },
-    ]
+pub struct SoftwareClient<'a> {
+    pub connection: Connection,
```

Comment:
still not sure why we need that connection in struct and even why it needs to be public

Reply:
For no particular reason (for sure it does not need to be public). We need to think about the proper handling of connections (so you can set the address through the CLI and so on). And then we can evaluate what is better.

But I will remove it for now.

Reply:
I have dropped the connection for this client.